### PR TITLE
Tsk 1089

### PR DESCRIPTION
**Pull request by BerndBreier**

### CODE REVIEW

File: taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/service/CamundaTaskEventsService.java [hunk 1]
```diff
@@ -32,8 +32,21 @@
  */
 public class CamundaTaskEventsService {
```
Comment (gitgoodjhe) [inline taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/service/CamundaTaskEventsService.java]: Can't we use an interface for the properties here again?

File: taskana-adapter/src/main/java/pro/taskana/adapter/impl/TaskanaTaskStarter.java [hunk 2]
```diff
@@ -28,7 +28,8 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TaskanaTaskStarter.class);
 
-  @Autowired AdapterManager adapterManager;
+  @Autowired
+  AdapterManager adapterManager;
```
Comment (gitgoodjhe) [inline taskana-adapter/src/main/java/pro/taskana/adapter/impl/TaskanaTaskStarter.java]: something seems off with the formatter again

File: taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/TaskanaConfigurationProperties.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package pro.taskana.adapter.camunda;
+
+public interface TaskanaConfigurationProperties {
+
+  static final String TASKANA_ADAPTER_CREATE_SCHEMA = "taskana.adapter.create_outbox_schema";
```
[the remark this answers is not in the document]
Comment (BerndBreier) [reply taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/TaskanaConfigurationProperties.java]: done

File: taskana-adapter/src/main/java/pro/taskana/adapter/impl/TaskanaTaskStarter.java (hunk 2 above)
Comment (BerndBreier) [reply taskana-adapter/src/main/java/pro/taskana/adapter/impl/TaskanaTaskStarter.java]: done

File: taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/parselistener/TaskanaParseListenerProcessEnginePlugin.java [hunk 4]
```diff
@@ -48,27 +59,89 @@ private void initParseListeners(ProcessEngineConfigurationImpl processEngineConf
...
-    outboxSchema = (outboxSchema == null || outboxSchema.isEmpty()) ? DEFAULT_SCHEMA : outboxSchema;
-
-    TaskanaOutboxSchemaCreator schemaCreator = new TaskanaOutboxSchemaCreator(dataSource,
-        outboxSchema);
-    try {
-      schemaCreator.run();
-    } catch (Exception e) {
-      LOGGER.warn("Caught {} while trying to initialize the outbox-table", e);
-      // processEngineConfiguration.getProcessEngine().close();
+    String outboxSchema = ReadPropertiesHelper.getPropertyValueFromFile(TASKANA_OUTBOX_PROPERTIES,
```
[the remark this answers is not in the document]
Comment (BerndBreier) [reply taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/parselistener/TaskanaParseListenerProcessEnginePlugin.java]: extracted method createSchema from initOutbox ...

**Pull request merged**